ts/features/bonus/siciliaVola/screens/voucherGeneration/SelectBeneficiaryCategoryScreen.tsx

```diff
@@ -14,25 +15,64 @@ import {
   svGenerateVoucherSelectCategory
 } from "../../store/actions/voucherGeneration";
 import FooterWithButtons from "../../../../../components/ui/FooterWithButtons";
-import ButtonDefaultOpacity from "../../../../../components/ButtonDefaultOpacity";
 import { SvBeneficiaryCategory } from "../../types/SvVoucherRequest";
 import { selectedBeneficiaryCategorySelector } from "../../store/reducers/voucherRequest";
 import {
   navigateToSvDisabledAdditionalInfoScreen,
+  navigateToSvKoSelectBeneficiaryCategoryScreen,
   navigateToSvSickCheckIncomeThresholdScreen,
   navigateToSvStudentSelectDestinationScreen,
   navigateToSvWorkerCheckIncomeThresholdScreen
 } from "../../navigation/actions";
 import I18n from "../../../../../i18n";
+import { RadioButtonList } from "../../../../../components/core/selection/RadioButtonList";
+
+type BeneficiaryCategory = SvBeneficiaryCategory | "other";
 
 type Props = ReturnType<typeof mapDispatchToProps> &
   ReturnType<typeof mapStateToProps>;
 
+const getCategoryBeneficiaryItems = (): ReadonlyArray<{
+  label: string;
+  id: BeneficiaryCategory;
+}> => [
```

Comment:
```suggestion
const getCategoryBeneficiaryItems = (): ReadonlyArray<
  RadioItem<BeneficiaryCategory>
> => [
```
by exporting RadioItem you'll have a more compact type declaration

Reply:
Surely better! Thank you! 
Updated in [3bf3fb7](https://github.com/pagopa/io-app/pull/3273/commits/3bf3fb7533bed5cdb9e7e84f42b2713f377ea343)